modules/nw-sriov-dpdk-running-testpmd.adoc

```diff
@@ -0,0 +1,20 @@
+// Module included in the following assemblies:
+//
+// * networking/hardware_networks/using-dpdk-and-rdma.adoc
+
+:_content-type: REFERENCE
+[id="nw-sriov-dpdk-running-testpmd_{context}"]
+= Example testpmd script
+
+The following is an example script for running `testpmd`:
+
+[source,yaml]
+----
+    set -ex
```

Comment:
If this is a Bash script, it could start with `#!/bin/bash'. Also, you could remove the indentations.

For a potential later PR, you could consider separating the lines in this script into individual procedure steps in their own code blocks, with descriptions for each line. But, this does not need to hold up this current PR.

Reply:
Point noted for individual procedure steps. Indentations removed. Good spot on the shell script, added #!/bin/bash.